### ISSUE TITLE
Can't find packages

### ISSUE THREAD

**Issue by neumachen** [awslabs/golang-deployment-pipeline#4]
I'm having issues with the CodeBuild not finding packages even though my packages are already vendored.

**Comment by tebruno99** [comment]
Yep, same problem here.  It seems that CodeBuild has automation that forms the checkout from git in the correct module/dir structure. However CodePipe line just drops a zip of the repo into the src directory without any of the structure so modules cannot be found.

**Comment by forstmeier** [comment]
Has a solution been found for this? I can't seem to set up a clean/simple way to handle a project that has subpackages. E.g.
```
parentdir/
|_ childdir/
|  |_ child.go
|_ main.go
```
Where the `main.go` file includes `import "parentdir/childdr"`. I'm running into issues where the `CODEBUILD_SRC_DIR` doesn't seem to be a part of the `GOPATH` and then when it's added, those local packages are missing from the file path. Any insight/ideas would be super helpful.

**Comment by tebruno99** [comment]
@forstmeier, in our build scripts we now rsync the src folder into the GOPATH so go build can find our own modules. (Including the vendor directory)

**Comment by forstmeier** [comment]
@tebruno99 cool, thanks! I'm trying to do something like that in the `buildspec.yml` file but with little success. Do you have an example?

**Comment by tebruno99** [comment]
```phases:
  install:
    commands:
      - echo CODEBUILD_SRC_DIR - $CODEBUILD_SRC_DIR
      - echo GOPATH - $GOPATH
      - echo GOROOT - $GOROOT
      - echo Copying Source to Go module dir
      - ls -l $CODEBUILD_SRC_DIR
      - mkdir -p /go/src/github.com/tebruno99/lambda-auth
      - cp -a $CODEBUILD_SRC_DIR/* /go/src/github.com/tebruno99/lambda-auth/

  build:
    commands:
      - echo Build started on `date`
      - mkdir -p build
      - GOOS="linux" go build -o build/myHandler```
